submodules/README.md

```diff
@@ -10,6 +10,7 @@ This data can also be gathered by running `git submodule`.
 |------------------|--------------|:-----------:|:------------:|
 | base32           | d08059d      | 2019-06-20  | Yes (Forked) |
 | bitblob          | v1.1.4       | 2019-08-30  | Yes (Owned)  |
+| d2sqlite3        | 79bbf5f      | 2020-07-22  | Yes (Forked) |
```

Comment:
```suggestion
| d2sqlite3        | 8bd3a7b      | 2020-07-23  | Yes (Forked) |
```

This is the last commit of Master Branch. Is that right?
Last commits on Jul 23, 2020
The manual update log is tricky.